makefile

```diff
@@ -5,17 +5,22 @@ DMD_FLAGS := -O -inline $(DMD_COMMON_FLAGS)
 DMD_TEST_FLAGS := -unittest -g $(DMD_COMMON_FLAGS)
 LDC_FLAGS := -g -w -oq $(INCLUDE_PATHS)
 
-.PHONY: dmd ldc test
+.PHONY: dmd ldc gdc test
 
 dmd: bin/dfmt
 
 ldc: $(SRC)
-	ldc2 $(LDC_FLAGS) $(SRC) -ofbin/dfmt
+	ldc2 $(LDC_FLAGS) $^ -ofbin/dfmt
 	-rm -f *.o
 
-test:
-	dmd $(DMD_TEST_FLAGS) $(SRC) -ofbin/dfmt
+gdc: $(SRC)
+	gdc $(LDC_FLAGS) $^ -ofbin/dfmt
```

Comment:
I wasn't aware that LDCs flags were valid for GDC. I know that some are, but it might be better to introduce a new variable for this.